contrib/scoverage/worker/1.3.1/src/ScoverageReportWorkerImpl.scala

```diff
@@ -17,7 +17,7 @@ class ScoverageReportWorkerImpl extends ScoverageReportWorkerApi {
     val coverage = deserialize(coverageFileObj)
     coverage(invoked(findMeasurementFiles(dataDir)))
     val sourceFolders = sources.map(_.path.toIO)
-    val folder = new java.io.File(s"${selfDir}/${reportType.folderName}")
+    val folder = new java.io.File(s"${selfDir}/${reportType.folderName}/dest")
```

Comment:
Best would be to take a `implicit ctx: Ctx.Dest` parameter in the `def buildReport` instead of the explicit `selfDir`. Then you can simply use `ctx.dest` as target folder. This would made the caller side easier too.

Reply:
@nvander1 I opened a PR to your PR some time ago...